ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-ospf.j2

```diff
@@ -27,6 +27,42 @@ router ospf {{ process_id }}
    bfd default
 {%     endif %}
 {%     for area in router_ospf.process_ids[process_id].areas | arista.avd.natural_sort %}
+{# OSPF stub area configuration #}
+{%         if router_ospf.process_ids[process_id].areas[area].type | arista.avd.default("normal") | lower == "stub" %}
```

Comment:
In `eos_cli_config_gen` we typically don't don't normalize the string with  `| lower` - was there a specific reason in this use case?
I would have expected simply this: ` {%  if router_ospf.process_ids[process_id].areas[area].type | arista.avd.defined("stub") %}`